docs/DeveloperGuide.adoc

```diff
@@ -313,7 +334,55 @@ _{More to be added}_
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `TeaPet` and the *Actor* is the `Teacher`, unless specified otherwise)
+
+[discrete]
+=== Use case: Add student
+
+*MSS*
+
+1.  User keys in a student
+2.  TeaPet adds the student to the class list
++
```

Comment:
Maybe you can add the UI changes done by TeaPet as a 3rd step for MSS. (e.g. TeaPet shows the added student).